lib/features/pokemon_details/widgets/about_item.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:pokedex/widgets/app_text_widgets.dart';
+import 'package:pokedex/widgets/spacings.dart';
+
+class AboutItem extends StatelessWidget {
+  const AboutItem({
+    super.key,
+    required this.itemLabel,
+    required this.itemContent,
+  });
+
+  final String itemLabel;
+  final String itemContent;
+
+  @override
+  Widget build(BuildContext context) => Column(
+        children: [
+          Row(
+            children: [
+              SizedBox(
+                width: MediaQuery.of(context).size.width - 250.0,
```

Comment:
kindly declare `MediaQuery.of(context).size.width` once then reuse it instead of calling it twice 🙂